src/Microsoft.ML.Legacy/PredictionModel.cs

```diff
@@ -22,6 +21,7 @@ public class PredictionModel
         internal PredictionModel(Stream stream)
         {
             _env = new ConsoleEnvironment();
+            AssemblyRegistration.RegisterAssemblies(_env);
```

Comment:
>RegisterAssemblies [](start = 33, length = 18)

do we need the same thing fir `TransformModel`?

Reply:
This is only needed for Legacy namespace.

---
In reply to: [233615020](https://github.com/dotnet/machinelearning/pull/1570#discussion_r233615020) [](ancestors = 233615020)